shelly/src/main/java/au/com/jtribe/shelly/Email.java

```diff
@@ -99,9 +99,8 @@ public Email body(String body) {
      * @return Boolean true if there is an activity to handle the Intent and it was started, false otherwise.
      */
     public boolean send() {
-        Intent emailIntent = new Intent(Intent.ACTION_SENDTO);
+        Intent emailIntent = new Intent(Intent.ACTION_SEND);
```

Comment:
I think we want to continue using `ACTION_SENDTO` because an Email will typically be sent to a particular person.

From the docs: `Activity Action: Send a message to someone defined by the data.` vs. `ACTION_SEND` which is a generic send without necessarily specifying who should receive it.